Make braille and vision follow caret in browse mode (#20465)
nvaccess/nvda: user_docs/en/changes.md

* NVDA should no longer cause File Explorer or other applications to crash when NVDA is exited or restarted. (#16207)
* Focus is no longer silent on list items in Qt-based applications (such as Telegram Desktop) when the item exposes the UIA SelectionItem pattern without an associated action interface. (#20255, @rezabakhshilaktasaraei)
* The HID keyboard input simulation setting for ALVA braille displays is now remembered across reconnects and restarts. (#20455, @Cary-rowen)

> * Braille and the magnifier now follow the spoken text during say all in browse mode and other virtual buffers, not just when braille is tethered to review. (#3287, @LeonarddeR)

Reviewer: Was there an issue with Magnifier?
In 2026.2beta6, It seems to me that it already follows in browse mode say all, even if review does not follow caret.
Note that Magnifier is not tied to vision framework though.
For visual highlighter (of browse mode cursor), I am not able to figure if this PR changes anything but I would expect yes.

It would be nice to confirm with a well sighted person what was working or not before and after this PR to put a correct change log regarding Magnifier / Visual highlighter.
Maybe @Qchristensen? (beware to disable caret move review cursor with NVDA+6 when testing).

Author: This is a good point. As one of the authors of the vision framework, I had forgotten that magnifier operates entirely separate.
That said, I'm pretty sure that this will ensure that the highlighter will now follow during say all while it probably didn't before, but that's also something a sighted someone should verify.
May be for now, we'll scope the changelog to the actual issue it closes, regarding Braille.
```suggestion
* Braille now follows the spoken text during say all in browse mode when braille is tethered to focus. (#3287, @LeonarddeR)
```
